src/main/java/com/google/alpollo/Song.java

```diff
@@ -0,0 +1,25 @@
+package com.google.alpollo;
+
+public class Song {
+    private final String artist;
+    private final String name;
+    private final String album;
+
+    public Song(String artist, String name, String album) {
+        this.artist = artist;
+        this.name = name;
+        this.album = album;
+    }
+
+    public String artist() {
```

Comment:
Getters and setter should be called like 'getArtist'/'setArtist' to make obvious what this method does.